### PR TITLE
I added the real steps to add a bot to a Team in Microsoft teams.

### PR DESCRIPTION
**Pull request by DamienDoumer**

### CODE REVIEW

**Comment by scanum** [comment]
@LolaJacobsen Hi Lola, could you please take a look at this PR. Please merge if you agree with changes.

**Comment by LolaJacobsen** [comment]
Luis Carrasco is the PM for this article. I've ping'd him to review this PR.

File: Teams/add-bots.md [hunk 1]
```diff
@@ -60,7 +60,15 @@ Side load your bot for channels
...
-3. Select **Manage team** and then select the **Apps** tab.
-4. At the bottom-right of your screen, select **Upload a custom app**.
+2. Add **[App Studio](https://docs.microsoft.com/en-us/microsoftteams/platform/get-started/get-started-app-studio)**, app to Microsoft Teams.
+3. In App Studio, select the **Manifest Editor** Tab. 
+![Manifest Editor Tab Screenshot.](media/Adding_Bot_To_Teams.png)
+4. To add your bot, In capabilities, select bot and chose to add an existing bot, then you will have the option 
+to chose an existing bot from a drop or enter the Id of one of your existing bots.
+![Select your bot you already created.](media/Select_Existing_Bot.png)
 5. Browse to the location of your app package, select it, and then click **Open**.
+6. Select your bot's name (Don't forget to check the "Team" checkbox under the scope section)
```
Comment (nenonix) [inline Teams/add-bots.md]:
```suggestion
6. Select your bot's name (Don't forget to check the "Team" checkbox under the scope section).
```

Comment (DamienDoumer) [reply Teams/add-bots.md]: Ok.

**Comment by LolaJacobsen** [comment]
Ping'd Luis Carrasco again today.

**Comment by LolaJacobsen** [comment]
Larry Jin is now the PM for this feature. I've ping'd him.

**Comment by LolaJacobsen** [comment]
@larryjin Please review this PR. Let me know whether it's OK to merge.

File: Teams/add-bots.md [hunk 2]
```diff
@@ -60,7 +60,15 @@ Side load your bot for channels
...
-2. Select **More** (...) next to the team name.
-3. Select **Manage team** and then select the **Apps** tab.
-4. At the bottom-right of your screen, select **Upload a custom app**.
+2. Add **[App Studio]**(https://docs.microsoft.com/en-us/microsoftteams/platform/get-started/get-started-app-studio), app to Microsoft Teams.
+3. In App Studio, select the **Manifest Editor** Tab. 
+![Manifest Editor Tab Screenshot.](media/Adding_Bot_To_Teams.png)
+4. To add your bot, In capabilities, select bot and chose to add an existing bot, then you will have the option 
+to chose an existing bot from a drop or enter the Id of one of your existing bots.
+![Select your bot you already created.](media/Select_Existing_Bot.png)
 5. Browse to the location of your app package, select it, and then click **Open**.
```
Comment (larryjin) [inline Teams/add-bots.md]: Why is this step necessary? If you are already creating an app from scratch, you don't need to import anything existing, right?

Comment (DamienDoumer) [reply Teams/add-bots.md]: You are Right.

**Pull request merged**